tinker/faculty_bios/faculty_bio_controller.py

```diff
@@ -122,21 +124,201 @@ def _iterate_child_xml(self, child, author):
+                # job_title
+                for jobs in soup.findAll('job-titles'):
+                    max_jobs += 1
+                    school = jobs.find('school').text
+                    department = ""
+                    title = ""
+                    if school == 'Bethel University':
+                        department = ""
+                        title = jobs.find('job_title').text
+                    elif school == 'College of Arts and Sciences':
+                        department = jobs.find('department').text
+                        if jobs.find('department-chair').text == 'Yes':
+                            title = 'Department Chair'
+                        else:
+                            title = jobs.find('job_title').text
+                    elif school == 'College of Adult and Professional Studies':
+                        department = jobs.find('adult-undergrad-program').text
+                        if jobs.find('program-director').text == 'Yes':
+                            title = 'Program Director'
+                        else:
+                            title = jobs.find('job_title').text
+                    elif school == 'Graduate School':
+                        department = jobs.find('graduate-program').text
+                        if jobs.find('program-director').text == 'Yes':
+                            title = 'Program Director'
+                        else:
+                            title = jobs.find('job_title').text
+                    elif school == 'Bethel Seminary':
+                        department = jobs.find('seminary-program').text
+                        if jobs.find('lead-faculty').text == 'Other':
+                            title = jobs.find('job_title').text
+                        elif jobs.find('program-director').text == "Yes":
+                            title = "Program Director"
+                        elif jobs.find('lead-faculty').text != "Other":
+                            title = jobs.find('lead-faculty').text
+                    else:
+                        school = ""
+                        department = ""
+                        title = ""
+                    # Creates a dictionary with the relevant information
+                    my_dict = {
+                        'school' + str(max_jobs): school,
+                        'department' + str(max_jobs): department,
+                        'job_title' + str(max_jobs): title
+                    }
+                    # Adds the dictionary to the list
+                    my_list.append(my_dict)
+
+                # max_edu will hold the maximum number of jobs someone has
+                max_edu = 0
+                for edu in soup.findAll('education'):
+                    max_edu += 1
+                    # Creates the dictionary with the relevant information
+                    my_dict = {
```

Comment:
i know i mentioned using a dictionary for this. But couldn't you just add these to your list on the fly?

Reply:
Added these to the main dictionary on the fly.